### PR TITLE
Added support for Linode DNS API.

### PR DESCRIPTION
**Pull request by trancee**

### CODE REVIEW

**Comment by Neilpang** [comment]
@trancee 
Please fix the issues first: https://travis-ci.org/Neilpang/acme.sh/jobs/192285276

Is it possible to implement without linode dependency ?

**Comment by trancee** [comment]
I will have to rewrite the implementation to use the API directly which might take some time. But I can have a look into it.

File: dnsapi/README.md [hunk 1]
```diff
@@ -257,6 +257,25 @@ acme.sh --issue --dns dns_ad -d example.com -d www.example.com
 The `AD_API_KEY` will be saved in `~/.acme.sh/account.conf` and will be reused
 when needed.
 
+## 14. Use Linode domain API
+
+You will need to install the Linode CLI and set it up accordingly.
```
Comment (Neilpang) [inline dnsapi/README.md]: Please update the readme also.

File: dnsapi/dns_linode.sh [hunk 2]
```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env bash
```
Comment (Neilpang) [inline dnsapi/dns_linode.sh]: use:
```
#!/usr/bin/env sh
```

**Pull request merged**